client/my-sites/exporter/post-type-options.jsx

```diff
@@ -69,28 +74,39 @@ const PostTypeOptions = React.createClass( {
 	},
 
 	renderFields() {
-		const { options } = this.props;
+		const {
+			options,
+			siteId,
+			postType,
+			values,
+		} = this.props;
```

Comment:
This isn't important, and isn't a blocker, but since we are already rearranging things here, could we alphabetize this list?